services/transition/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: '3.7'
+
+x-transition: &transition
+  build:
+    context: .
+    dockerfile: Dockerfile.govuk-base
+  image: transition
+  volumes:
+    - ${GOVUK_ROOT_DIR:-~/govuk}:/govuk:delegated
+    - home:/home/build
+  working_dir: /govuk/transition
+
+services:
+  transition-lite:
+    <<: *transition
+    depends_on:
+      - nginx-proxy-app
+      - postgres
+      - redis
+      - transition-worker
+    environment:
+      DATABASE_URL: "postgresql://postgres@postgres/transition"
+      TEST_DATABASE_URL: "postgresql://postgres@postgres/transition-test"
+
+  transition-app: &transition-app
+    <<: *transition
+    depends_on:
+      - nginx-proxy-app
+      - postgres
+      - redis
+      - transition-worker
+    environment:
+      DATABASE_URL: "postgresql://postgres@postgres/transition"
+      REDIS_URL: redis://redis
+      VIRTUAL_HOST: transition.dev.gov.uk
+      HOST: 0.0.0.0
+    ports:
+      - "3000"
+    # Change to 'bin/rails --restart' when rails >= 5.2
+    command: /bin/sh -c "rm -f tmp/pids/server.pid && rails s"
+
+  transition-worker:
+    <<: *transition
+    depends_on:
+      - postgres
+      - redis
+      - nginx-proxy-app
```

Comment:
We shouldn't need `nginx-proxy-app` here, as it only applies to web app services.